bcda/models/models.go

```diff
@@ -149,6 +149,7 @@ func (job *Job) GetEnqueJobs(resourceTypes []string, since string, newBeneficiar
 					ResourceType:    rt,
 					Since:           since,
 					TransactionTime: job.TransactionTime,
+					Priority:        setJobPriority(*aco.CMSID, rt, len(since) != 0),
```

Comment:
According to [this documentation](https://godoc.org/github.com/bgentry/que-go#Job), `Priority` should not be set as part of the `Args` field.  Instead, it is its own field and should be set independently [here](https://github.com/CMSgov/bcda-app/blob/master/bcda/models/models.go#L157-L160).